src/spark_config/config.py

```diff
@@ -213,7 +216,9 @@ def load(cls, filepath, strict=True, warn_missing=False):
         instance = cls()
         with pathlib.Path(filepath).open("r") as fin:
             instance.update(
-                yaml.safe_load(fin), strict=strict, warn_missing=warn_missing
+                yaml.load(fin),
+                strict=strict,
+                warn_missing=warn_missing,
```

Comment:
Did you accidentally force the line break? I think if you drop the last comma it should go back